.env

```diff
@@ -1,5 +1,4 @@
 COMPOSE_DOCKER_CLI_BUILD=1
-COMPOSE_FILE=.docker-compose.yml
```

Comment:
Do I understand correctly that the removed lines in this diff were redundant, in that they specified default values?

Reply:
No. The filenames were changed from non-default values to default values.
- `.docker-compose.yml` -> `docker-compose.yml` (leading period removed)
- `.dockerfiled` -> `Dockerfile`